internal/handlers/common/update.go

```diff
@@ -148,7 +155,10 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 
 		default:
 			if strings.HasPrefix(updateOp, "$") {
-				return false, NewCommandError(ErrNotImplemented, fmt.Errorf("UpdateDocument: unhandled operation %q", updateOp))
+				return false, commonerrors.NewCommandError(
```

Comment:
I see codecov complaints, but as this is rather a refactoring, not a new implementation, I guess we don't add new tests for codecov here.

Reply:
Yes and we don't test `ErrNotImplemented` so I would say it's okay